index.html

```diff
@@ -154,10 +154,15 @@ <h4>This is an aside</h4>
 					<h2>Lists</h2>
 				</header>
 				
-					<h3>Definition list</h3>
+					<h3>Description list</h3>
```

Comment:
I'm curious, why was the name of this element changed? I think "definition list" is the canonical/original term — and btw, AFAIK the "t" and "d" in `<dt>` and `<dd>` stand for "term" and "definition" (or "description"), rather than "title" and "division".

In particular, the use of "division" seems rather confusing to me, whereas "title" and "description list", albeit uncommon, are at least intelligible.

Reply:
I think it changed in the spec at some point from Definition to Description, which is what it's called on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) and in the [spec](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element). Maybe to differentiate from the definition element `dfn`.

As for the title/division part, you're correct. The proper names appear to be Description Term and Description Details. I'll get those updated.